### PR TITLE
Remember logged out mp-jwts

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt_fat

File: dev/com.ibm.ws.security.mp.jwt/resources/com/ibm/ws/security/mp/jwt/resources/MicroProfileJwtMessages.nlsprops [hunk 1]
```diff
@@ -150,3 +150,8 @@ MP_JWT_FRONT_END_ERROR.useraction=Contact the system administrator to resolve th
 MPJWT_NOT_FOUND_IN_APPLICATION=CWWKS5526W: The MicroProfile JWT feature cannot perform authentication because it is expecting [{0}] authentication type in the application, but found [{1}]. The [{2}] attribute is set to [{3}]. 
 MPJWT_NOT_FOUND_IN_APPLICATION.explanation=To perform authentication successfully, do one of the following. a) Make sure that the ignoreApplicationAuthMethod attribute is "true" b) loginConfig annotation is set to MP-JWT in the application.
 MPJWT_NOT_FOUND_IN_APPLICATION.useraction=Ensure that the server or application configuration is updated.
+
+# do not translate HttpServletRequest.logout
+JWT_PREVIOUSLY_LOGGED_OUT=CWWKS5527E: The MicroProfile JWT feature cannot perform authentication because the JWT in the request was previously logged out.
+JWT_PREVIOUSLY_LOGGED_OUT.explanation=An HttpServletRequest.logout() call was previously made for a subject that was authenticated with this JWT.
```
Comment (helyarp) [inline dev/com.ibm.ws.security.mp.jwt/resources/com/ibm/ws/security/mp/jwt/resources/MicroProfileJwtMessages.nlsprops]: The "was previously made" is not optimal. Maybe change the explanation to:
A previous HttpServletRequest.logout() call was made for a subject that was authenticated with this JWT.

Comment (brutif) [reply dev/com.ibm.ws.security.mp.jwt/resources/com/ibm/ws/security/mp/jwt/resources/MicroProfileJwtMessages.nlsprops]: fixed

**Comment by brutif** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt_fat

**Comment by brutif** [comment]
pb is good

**Comment by brutif** [comment]
#8593 is opened to add FAT test

**Pull request merged**